nagios/manifest.json

```diff
@@ -11,6 +11,10 @@
   "manifest_version": "1.0.0",
   "metric_prefix": "nagios.",
   "name": "nagios",
+  "process_signatures": [
+    "service snmpd restart",
+    "systemctl restart snmpd.service"
```

Comment:
Is this correct/unique?

Reply:
It's not thanks for pointing this out, will replace by nagios